src/helpers/BapiClient.ts

```diff
@@ -477,9 +483,17 @@ export class BapiClient {
       );
 
       for (const itemToAdd of items) {
-        const existingBasketItem = client.latestBasket.items.find(
-          item => item.variant.id == itemToAdd.variantId,
-        );
+        const existingBasketItem = client.latestBasket.items.find(item => {
+          if (item.variant.id != itemToAdd.variantId) {
+            return false;
+          }
+
+          if (!options.considerItemGroupForUniqueness) {
```

Comment:
I like the option but technically I think we don't even need it.

For tenants who don't use the item they behaviour afaik shouldn't change, e.g. it would always be `null` so it would match

Reply:
I checked with Max and he wasn't sure and we decided on adding the flag